problems/5.loops.js

```diff
@@ -2,32 +2,55 @@
 * @param {number} num - The number to iterate up to
 * @param {number[]} - An array of even numbers from 0 to {num}
 */
-function evenNumbers() {
-
+function evenNumbers(num) {
+  let arr = []
+  let num1 = 0
+  while (num1 <= num) {
+    if (num1 % 2 === 0) {
+      arr.push(num1)
+    }
+    num1 += 1
+  }
+  return arr
 }
 
 /** Returns all multiples of a given number up to 1000
  * Example if num was 3 the output would be [0, 3, 6, 9, 12, ..., 999]
 * @param {number} num - The number to find multiples of
 * @returns {number[]} - An array of all multiples of {num} up to 1000
 */
-function multiplesToOneThousand() {
+function multiplesToOneThousand(num) {
+  let arr = []
+  let num1 = 0
+  while (num1 <= 1000) {
+    if (num % num1 === 0) {
+      arr.push(num1)
+    }
 
+    num1+= num
```

Comment:
You want to increment num1 by 1, not by the parameter. If I enter in the parameter as 1000, it'll immediately jump from 0 => 1000. I think I understand your approach but it needs some refactoring.